Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/stats/EventAttributeValue.java

```diff
@@ -132,37 +155,34 @@ public double getOppCases() {
     }
 
     public IntArrayList getOccurCaseIndexes() {
-        if (occurCaseIndexes == null) {
-            int[] array = occurCasesIndexSet.stream().mapToInt(x -> x).toArray();
-            occurCaseIndexes =new IntArrayList(array);
-        }
-        return occurCaseIndexes;
+        return new IntArrayList(getOccurCasesIndexSet().stream().mapToInt(x->x).toArray());
     }
 
+    @Override
     public double getPercent() {
-        return percent;
+        return 100 * ((double) getOccurCasesIndexSet().size() / totalCases);
     }
```

Comment:
Just like in CaseAttributeValue, I think this ought to be "getFrequency" without being a percentage, and the current "getFrequency" ought to be `formatAsPercent(getFrequency())`.